ErrorLog: started @ 2020-02-24
22:13:11:741 : Importing Error: File testRow\b\b.par failed to import: 
	Message : java.io.IOException,null
	[edu.carleton.enchilada.dataImporters.ATOFMSDataSetImporter.processDataSet(ATOFMSDataSetImporter.java:287), edu.carleton.enchilada.dataImporters.ATOFMSDataSetImporter.collectRowInfo(ATOFMSDataSetImporter.java:224), edu.carleton.enchilada.dataImporters.ATOFMSDataSetImporterTest.testCollectTableInfo(ATOFMSDataSetImporterTest.java:174), sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method), sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62), sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43), java.lang.reflect.Method.invoke(Method.java:498), junit.framework.TestCase.runTest(TestCase.java:176), junit.framework.TestCase.runBare(TestCase.java:141), junit.framework.TestResult$1.protect(TestResult.java:122), junit.framework.TestResult.runProtected(TestResult.java:142), junit.framework.TestResult.run(TestResult.java:125), junit.framework.TestCase.run(TestCase.java:129), junit.framework.TestSuite.runTest(TestSuite.java:255), junit.framework.TestSuite.run(TestSuite.java:250), org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:84), org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365), org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273), org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238), org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159), org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384), org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345), org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126), org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)]
